### PR TITLE
[MTO] Added mattermost use case

### PR DESCRIPTION
**Pull request by hussnain612**
Added a use case for Mattermost in MTO docs

### CODE REVIEW

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-50-11a77403`

File: docs/content/sre/tenant-operator/usecases/mattermost.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+### Creating Mattermost Teams for your tenant
+
+Bill wants each tenant to also have their own Mattermost Teams. To Make sure this happens correctly, Bill will first add the `stakater.com/tenant` label to the tenant:
```
Comment (MahnoorAsghar) [inline docs/content/sre/tenant-operator/usecases/mattermost.md]: 'To make sure' instead of 'To Make sure'

Comment (MahnoorAsghar) [inline docs/content/sre/tenant-operator/usecases/mattermost.md]: Do you mean 'stakater.com/mattermost:true' label?

Comment (hussnain612) [reply docs/content/sre/tenant-operator/usecases/mattermost.md]: Updated

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-50-73974656`

**Review by MahnoorAsghar — APPROVED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.26`

**Pull request merged**